### PR TITLE
Basic control structures

### PR DESCRIPTION
**Pull request by nachosoloaga**
This PR includes slides for basic control structures, covering:

- If/else
- While
- Do while
- For
- Switch

### CODE REVIEW

File: src/slides/class1.jsx [hunk 1]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+          {`
+            if (condition) {
+              // if block statements
+            } else {
+              // else block statements
+            }
+          `}
+        </CodePane>
+        <Text fontSize={30}>
+          La sentencia if evalua la expresión entre paréntesis y convierte su resultado a booleano.
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:677]:
```suggestion
          La sentencia if evalúa la expresión entre paréntesis y luego utiliza el valor de verdad del resultado de dicha evaluación.
```

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+          Por ejemplo, la siguiente condición nunca ejecutará el bloque de código <b>block1.</b>
+        </Text>
+        <CodePane language="js">
+            {
+              `
+                if (0) { // 0 es un valor falsy
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:683]: Lets use english for all the code snippets

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+          mucho más corta y simple.
+        </Text>
+        <CodePane language="js">
+          {`
+            let result = condition ? value1 : value2;
+          `}
+        </CodePane>
+        <Text fontSize={30}>
+          En caso que <b>condition</b> evalue a true, entonces se asignará <b>value1</b> a la variable <b>result</b>,
+          y caso contrario se asignará <b>value2</b>.
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:709]:
```suggestion
          caso contrario se asignará <b>value2</b>. 
```

File: src/slides/class1.jsx [hunk 4]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+      <div>
+        <Heading margin="0px" fontSize="50px">
+            Conditional operator (?) (cont...)
+          </Heading>
+        <Text fontSize={30}>
+          El <b>?</b> puede combinarse múltiples veces simulando sentencias <b>if/else</b>:
+        </Text>
+        <CodePane language="js">
+            {`
+              let carBrand = "Peugeot"
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:724]:
```suggestion
              let carBrand = "Peugeot";
```

File: src/slides/class1.jsx [hunk 5]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+
+              let message = (carBrand == "Suzuki") ? "Nice Suzuki!" :
+                (carBrand == "Fiat") ? 'Nice Fiat!' :
+                (carBrand == "Peugeot") ? 'Nice Peugeot!' :
+                'I don't know that brand, nice car!';
+              
+              alert( message );
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:731]:
```suggestion
              alert(message);
```

File: src/slides/class1.jsx [hunk 6]
```diff
@@ -625,5 +625,219 @@ export default () => (
...
+        <Text fontSize={30}>
+          Para finalizar la ejecución del loop antes que se cumpla la condición podemos utilizar la sentencia <b>break</b>.
+        </Text>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          For
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:787]: Can we add [for..in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in), [for..of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) and maybe [a comparison between them](https://www.typescriptlang.org/docs/handbook/iterators-and-generators.html#forof-vs-forin-statements)?

**Review by JulianPasquale — APPROVED** [review]

File: src/slides/class1.jsx (hunk 6 above)
Comment (nachosoloaga) [reply src/slides/class1.jsx:787]: Sounds good!

**Pull request merged**